protocol/x/perpetuals/keeper/perpetual.go

```diff
@@ -1222,6 +1234,30 @@ func (k Keeper) ModifyFundingIndex(
 	return nil
 }
 
+func (k Keeper) ModifyOpenInterest(
```

Comment:
Only used for test in this PR, but in a future will be used for actually updating open interest of a perp.